### DISCUSSION TITLE
Is it possible to setup postal on local machine to send and receive bulk mails

### DISCUSSION THREAD

**Discussion by pratik2630** [postalserver/postal#2447] [Help with using Postal]
I've checked multiple documentation but they are all focused on directly doing setup on server . I want to do it locally by making redhat/fedora virtualmachine as my server . Please guide me how should i do it , any suggestions , related documentation are much appreciated.

**Comment by sinnbeck** [comment]
Be aware that you can only use postal as a relay for incoming messages (redirecting to another email server, or web server). 

I would assume you can just set it up as if it was an external server. But you still need to point a domain to your local ip and make sure that all traffic on port 5000 and 25 are sent to your computer (in your firewall)

Also contact your internet provider and ask them if they can set up a PTR record for your IP to the domain. If not, your mail might end up in spam
